### ISSUE TITLE
Package issue:  sherpa-ncnn is outdated on piwheels

### ISSUE THREAD

**Issue by csukuangfj** [piwheels/packages#392]
### Package name

sherpa-ncnn

### Package version

all

### PyPI URL

https://pypi.org/project/sherpa-ncnn/

### piwheels URL

https://www.piwheels.org/project/sherpa-ncnn/

### Python version

- [X] Python 3.7
- [X] Python 3.9

### I am the maintainer

- [X] Yes

### More information

https://www.piwheels.org/simple/sherpa-ncnn/
hosts only sherpa-ncnn <= 1.2

Could you update it? We have sherpa-ncnn 2.1.3 now.

**Comment by bennuttall** [comment]
We are currently building the backlog for Python 3.11 so everything is on hold while we do that. New releases will be built for all versions once that backlog is finished.
